babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateFunctionSqlForm.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel;
+
+import org.apache.calcite.sql.SqlCreate;
+import org.apache.calcite.sql.SqlDataTypeSpec;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlSpecialOperator;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.util.ImmutableNullableList;
+
+import java.util.List;
+
+public class SqlCreateFunctionSqlForm extends SqlCreate {
+  public final SqlIdentifier functionName;
+  public final SqlIdentifier specificFunctionName;
+  public final List<SqlIdentifier> fieldNames;
+  public final List<SqlDataTypeSpec> fieldTypes;
+  public final SqlDataTypeSpec returnsDataType;
+  public final DeterministicType isDeterministic;
+  public final ReactToNullInputType canRunOnNullInput;
+  public final boolean hasSqlSecurityDefiner;
+  public final int typeInt;
+  public final SqlNode returnExpression;
+
+  private static final SqlSpecialOperator OPERATOR =
+      new SqlSpecialOperator("CREATE FUNCTION", SqlKind.CREATE_FUNCTION);
+
+  /** Creates a SqlCreateFunctionSqlForm.
```

Comment:
nit: The first comment line should be on the line after the "/**".

Reply:
Fix made in https://github.com/googleinterns/calcite/pull/79/commits/0ae2a1dacf8f672feaaae88448156fbd8470c66f .